src/System.Private.CoreLib/shared/System/BitOps.cs

```diff
@@ -8,31 +8,60 @@
 
 using Internal.Runtime.CompilerServices;
 
+// Some routines inspired by the Stanford Bit Widdling Hacks by Sean Eron Anderson:
```

Comment:
Moved this comment up since this class will eventually contain several similar methods inspired by this source.